.github/workflows/release-python.yaml

```diff
@@ -1,41 +0,0 @@
-name: Release Python SDK
```

Comment:
Ah, we'll want to keep this workflow around – it's what implements the maturin `extism_sys` publish!

Reply:
thanks for taking a look, I will un-remove that one!